Yosemite/Yosemite/Actions/CouponAction.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import Networking
```

Comment:
nit: this import shouldn't be needed if we create a `typealias` of `CouponReport` under `Model` in the Yosemite layer.

Reply:
Thank you for pointing that out! I missed it 😅 Added in 1fcfb8a76c41f604a569b8c6fd81553f7d1a550a.